Makefile

```diff
@@ -1,7 +1,13 @@
 .PHONY: test
 
+# Disable CGO by default.
+export CGO_ENABLED=0
+
 bin/ssm-env: *.go
-	CGO_ENABLED=0 go build -o $@ .
+	# Enforce static linking, and copy the timezone data into the runtime so we
+	# can in places like scratch images.
+	go build -ldflags '-extldflags "-static"' -o $@ .
 
 test:
-	go test -race $(shell go list ./... | grep -v /vendor/)
+	# Testing for race conditions requires CGO be enabled.
+	CGO_ENABLED=1 go test -race $(shell go list ./... | grep -v /vendor/)
```

Comment:
Kinda wondering if we shouldn't `export CGO_ENABLED`, but rather add a specific target for static builds and use an inline `CGO_ENABLED=0` there.

Reply:
Or just leave it as it was, since that was working fine?